CInclude/geoworks.h

```diff
@@ -497,7 +497,9 @@ typedef enum /* word */ {
     CIF_GEODEX,
     CIF_BITMAP,
     CIF_SOUND_SYNTH,
-    CIF_SOUND_SAMPLE
+    CIF_SOUND_SAMPLE,
+    CIF_ICON_LIST,

```

Comment:
Was icon list defined before in some other place? (if yes, where) Was it defined before "fax file page with ink"?